### PR TITLE
Set BROWSER_GTM_ID from source

### PR DESCRIPTION
**Pull request by giorgiosironi**
Paired with https://github.com/libero/sample-configuration/pull/77 but not urgent.

### CODE REVIEW

File: environments/unstable.env [hunk 1]
```diff
@@ -1,3 +1,4 @@
+export BROWSER_GTM_ID=
```
Comment (thewilkybarkid) [inline environments/unstable.env:1]: Shouldn't be needed?

Comment (giorgiosironi) [reply environments/unstable.env:1]: No, it has a default; but prefer being explicit.

File: scripts/deploy.sh [hunk 2]
```diff
@@ -40,8 +38,8 @@ fi
 # shellcheck source=environments/unstable.env
 source "${environment_variables_file}"
 
-for environment_variable_name in ENVIRONMENT_NAME PUBLIC_PORT_HTTP PUBLIC_PORT_HTTPS POPULATE_CONTENT_STORES; do
+for environment_variable_name in BROWSER_GTM_ID ENVIRONMENT_NAME PUBLIC_PORT_HTTP PUBLIC_PORT_HTTPS POPULATE_CONTENT_STORES; do
     environment="${environment} ${environment_variable_name}=${!environment_variable_name}"
 done
 
-ssh -o StrictHostKeyChecking=no -i "$key" "$ssh_hostname" ENVIRONMENT_NAME="${environment_name}" PUBLIC_PORT_HTTP="${public_port_http}" PUBLIC_PORT_HTTPS="${public_port_https}" "$environment" BRANCH_NAME="${branch_name}" /tmp/remote-deploy.sh
+ssh -o StrictHostKeyChecking=no -i "$key" "$ssh_hostname" "$environment" BRANCH_NAME="${branch_name}" /tmp/remote-deploy.sh
```
Comment (thewilkybarkid) [inline scripts/deploy.sh:45]: Unrelated changes?

**Review by thewilkybarkid — APPROVED** [review]
(Assume the value is correct!)

File: scripts/deploy.sh (hunk 2 above)
Comment (giorgiosironi) [reply scripts/deploy.sh:45]: Yes and no, `$environment` covers all those variables including `BROWSER_GTM_ID` now, so fewer surprises by removing what's redundant
